README.md

```diff
@@ -86,7 +86,7 @@ cd Grasscutter
 .\gradlew jar # Compile
 ```
 
-##### Linux
+##### GNU/Linux
```

Comment:
please remove `GNU/`

Reply:
Why?

Reply:
you can do something like Linux (GNU), but i think having the linux first makes more sense

Reply:
Done